use-cases/vehicle-routing-time-windows/src/main/java/org/acme/vehiclerouting/domain/Location.java

```diff
@@ -0,0 +1,55 @@
+package org.acme.vehiclerouting.domain;
+
+import java.util.Map;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+@JsonFormat(shape = JsonFormat.Shape.ARRAY)
+public class Location {
+
+    private double latitude;
+    private double longitude;
+
+    @JsonIgnore
+    private Map<Location, Long> drivingTimeMap;
```

Comment:
It's not clear that the Long of drivingTime is in seconds.

Proposal A) Add a javadoc on this field that it's in seconds
Proposal B) change field name to drivingTimeInSecondsMap
Proposal C) change Long to Duration.

Which of these are clear in the Swagger open API?

Reply:
I like either B) or C). Swagger has nothing to do with this field, as the distance matrix never gets on the client side.